### ISSUE TITLE
Add support to connect directly to chromecast via IP

### ISSUE THREAD

**Issue by kneedler** [keredson/gnomecast#50]
This is really cool. Thanks for making this!

For some reason I can't get chromecast discovery to work with my network/router, but I can figure out the IP and verify that ports 8008 and 8009 are open and I can query some info about it on port 8008 using curl.

I played around with the https://github.com/balloob/pychromecast library and can successfully build ChromeCast() object just using the IP. 

Can you add support for connecting to chromecast via IP? As in something like:

`gnomecast -d "192.168.1.100"`

Cheers

**Comment by keredson** [comment]
They sounds reasonable.  Possibly we should also expose that in the ui?  A
final option in the drop-down "custom host..."?

**Comment by arulrajnet** [comment]
Yes. We need this support. Since mDNS discovery is not suitable for all environment. See my comment https://github.com/balloob/pychromecast/issues/117#issuecomment-385570860

**Comment by keredson** [comment]
Yeah, agreed.  Shouldn't be a huge lift.  I'm traveling for work ATM.  If someone wants to take a stab at a pr, it'd be more then welcome.  If not prob this weekend I'd get a chance.
